html/images/os/extremeboss.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1" width="100" height="100"><g clip-rule="evenodd" fill="#6502a2" fill-rule="evenodd" stroke-width="1.46"><path d="M95 16.04V0H4.918v63.96l19.303-16.04V16.04z"/><path d="M82.131 47.92V31.882H43.286v16.04zM24.221 80H95V63.96H4.918z"/></g><text style="line-height:1.25;text-orientation:upright" x="20.998" y="102.9" transform="scale(1.03945 .96205)" font-size="25.268" letter-spacing="0" word-spacing="0" fill="#6502a2"><tspan x="20.998" y="102.9">BOSS</tspan></text></svg>
```

Comment:
You need to drop length/width and add viewbox so SVGs render properly in IE.  Thanks